Content.Server/Spider/SpiderSystem.cs

```diff
@@ -0,0 +1,90 @@
+using System.Linq;
+using Content.Server.Actions;
+using Content.Server.Popups;
+using Content.Shared.Spider;
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.MobState.Components;
+using Content.Shared.Doors.Components;
+using Content.Shared.Physics;
+using Content.Shared.Maps;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Physics;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Spider
+{
+    public sealed class SpiderSystem : EntitySystem
+    {
+        [Dependency] private readonly PopupSystem _popup = default!;
+        [Dependency] private readonly ActionsSystem _action = default!;
+        [Dependency] private readonly IMapManager _mapManager = default!;
+        [Dependency] private readonly IPrototypeManager _proto = default!;
+        [Dependency] private readonly IRobustRandom _robustRandom = default!;
+        [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<SpiderWebObjectComponent, ComponentStartup>(OnWebStartup);
+            SubscribeLocalEvent<SpiderComponent, ComponentStartup>(OnSpiderStartup);
+            SubscribeLocalEvent<SpiderComponent, SpiderWebActionEvent>(OnSpawnNet);
+        }
+
+        private void OnSpiderStartup(EntityUid uid, SpiderComponent component, ComponentStartup args)
+        {
+            var netAction = new InstantAction(_proto.Index<InstantActionPrototype>("SpiderWebAction"));
+            _action.AddAction(uid, netAction, null);
+        }
+
+        private void OnWebStartup(EntityUid uid, SpiderWebObjectComponent component, ComponentStartup args)
+        {
+            _appearance.SetData(uid,SpiderWebVisuals.Variant, _robustRandom.Next(1, 3));
+        }
+
+        private void OnSpawnNet(EntityUid uid, SpiderComponent component, SpiderWebActionEvent args)
+        {
+            if (args.Handled)
+                return;
+
+            var transform = Transform(uid);
+
+            if (!_mapManager.TryGetGrid(transform.GridUid, out var grid))
+                return;
+
+            args.Handled = true;
+
+            var coords = transform.Coordinates;
+
+            // Spawn web in center
+            if (!IsTileBlockedByWeb(coords))
+                Spawn(component.WebPrototype, coords);  
+
+            // Spawn web in other directions
+            for (var i = 0; i < 4; i++)
+            {
+                var direction = (DirectionFlag) (1 << i);
+                coords = transform.Coordinates.Offset(direction.AsDir().ToVec());
+                    
+                if (!IsTileBlockedByWeb(coords))
+                    Spawn(component.WebPrototype, coords);
+            }
```

Comment:
Nope. It's opposite. I tested it few times already. And I tested this in online in battle situations
1) Blocking spread because center is blocked is lame. You can mispredict position for example and cooldown long enough to not let you easy spam it. And web is not so strong to not allow covering tiles around you. If you want cover missing tiles this blocking only makes your building worse. In narrow maintenance corridors you already limited with how much you can cover.
2) Why web check needs to be removed if this will spawn web on already created?

Reply:
Okay, after testing this in-game i see what you were trying to do and this is valid. 
Do make a check so that when you fail to place webs, it doesnt use the action and a popup appears informing the user of this, something like "You can't place webs here! All cardinal directions already have webs!"